apps/src/StudioApp.js

```diff
@@ -828,10 +828,14 @@ StudioApp.prototype.displayFeedback = function(options) {
 };
 
 /**
- *
+ * Runs the tests and returns results.
+ * @param {boolean} levelComplete Was the level completed successfully?
+ * @param {Object} options
+ * @return {number} The appropriate property of TestResults.
  */
 StudioApp.prototype.getTestResults = function(levelComplete, options) {
-  return this.feedback_.getTestResults(levelComplete, options);
+  return this.feedback_.getTestResults(levelComplete,
+      this.checkForEmptyBlocks_, options);
```

Comment:
I was reluctant to modify options, since I don't know much about what it contains and we'd talked about wanting to treat it as `const`.  As for the @param tag, you'll find it in feedback.js above `FeedbackUtils.prototype.getTestResults`; I've added an argument to that method, I'm merely calling it here with the local `this.checkForEmptyBlocks_` value.